io/iouring-wrapper.cpp

```diff
@@ -501,21 +511,9 @@ class iouringEngine : public MasterEventEngine, public CascadingEventEngine {
         }
     }
 
-    void run_cancel_poller() {
-        while (m_cancel_poller_running) {
-            uint32_t poll_mask = evmap.translate_bitwisely(EVENT_READ);
-            int ret = async_io(&io_uring_prep_poll_add, -1, 0, m_cancel_fd, poll_mask);
-            if (ret < 0) {
```

Comment:
Should not use one-shot poll. There is a risk of missing eventfd notification, right before starting the next round of polling.